app/tasks.py

```diff
@@ -146,7 +151,12 @@ def import_product_db(db: Session, batch_size: int = 1000) -> None:
                         Product.updated == None,  # noqa: E711, E501
                     )
                 )
-                .values(**item)
+                .values(
+                    source=Flavor.off
```

Comment:
`source=item.get("source", Flavor.off)`

Reply:
actually the `item.get("source")` will always be None, as "source" is not in OFF_FIELDS

Reply:
so i would add this above in the "item" creation section
`item["source"] = Flavor.off`

Reply:
Ideed. I wanted a conditional in the values on `Product.source is None` but that did not work out so I thought `item` would do. Thought about using a case statement but not sure how that works in SQLAlchemy in the update method.